webservices/rest.py

```diff
@@ -43,6 +43,24 @@ def sqla_conn_string():
 api = restful.Api(app)
 db.init_app(app)
 
+# api.data.gov
+trusted_proxies = ('192.168.114.100',)
+PRODUCTION = os.getenv('PRODUCTION', False)
+print(PRODUCTION)
+
+@app.before_request
+def limit_remote_addr():
+    if PRODUCTION == 'true':
```

Comment:
Can we convert PRODUCTION to a boolean or at least a "truthy" value (like 1) instead of just checking for string equivalence? Even if the PRODUCTION value comes through as a string, it is likely to be set to "True", in which case this check would fail on the case difference, as well.